core/http/src/main/java/org/trellisldp/http/impl/RdfUtils.java

```diff
@@ -251,7 +247,7 @@ private static Boolean notCompareWithString(final RDFTerm term, final String str
      */
     public static IRI getProfile(final List<MediaType> acceptableTypes, final RDFSyntax syntax) {
         for (final MediaType type : acceptableTypes) {
-            if (RDFSyntax.byMediaType(type.toString()).filter(syntax::equals).isPresent() &&
+            if (RDFSyntax.byMediaType(type.toString()).filter(s -> s.equals(syntax)).isPresent() &&
```

Comment:
Why this change? Is `syntax` nullable?

Reply:
Yes, there are certain instances in the code where `syntax` will be `null`.